google/cloud/storage/tests/testbench.py

```diff
@@ -205,6 +205,34 @@ def update_acl(self, entity, role):
         """
         return self.insert_acl(entity, role)
 
+    def patch_acl(self, entity, request):
+        """
+        Patch a single AccessControl entry in this Object revision.
+
+        :param entity:str the name of the entity.
+        :param request:flask.Request the parameters for this request.
+        :return:dict with the contents of the ObjectAccessControl.
+        """
+        acl = self.get_acl(entity)
+        payload = json.loads(request.data)
+        request_entity = payload.get('entity')
+        if request_entity is not None and request_entity != entity:
+            raise ErrorResponse(
+                'Entity mismatch in ObjectAccessControls: patch, expected=%s, got=%s'
+                % (entity, request_entity))
+        etag_match = request.headers.get('if-match')
+        if etag_match is not None \
```

Comment:
The backslash-as-line-break is frowned upon in the style guide.  Could you use parens to allow regular line breaks instead?  Also, I usually see the logical operators placed *before* the line break, but don't really care as long as we're consistent throughout the code.

```
if (etag_match is not None and
        etag_match != acl.get('etag')):
    # logic_here
```

Reply:
Fixed (I hope).

> Also, I usually see the logical operators placed before the line break, but don't really care as long as we're consistent throughout the code.

I am letting [yapf](https://github.com/google/yapf) decide these things, so I expect them to be consistent (I am not enforcing python formatting in the CI builds *yet* though, so it might go wrong from time to time).  We can create a `.style.yapf` file if we do not like the defaults.

Reply:
I haven't been able to get yapf (or autopep8) to enforce a max line length within conditionals.  Yay for autoformatters that... don't autoformat.

As this starts to accumulate more and more python code, I'd like to add a pylintrc file.  Looks like the one published on the Tensorflow GitHub repo is pretty close to what we use for the internal linter:
https://github.com/tensorflow/tensor2tensor/blob/master/pylintrc